pkg/cmd/server/start/master_args.go

```diff
@@ -75,6 +78,7 @@ type MasterArgs struct {
 func BindMasterArgs(args *MasterArgs, flags *pflag.FlagSet, prefix string) {
 	flags.Var(&args.MasterAddr, prefix+"master", "The master address for use by OpenShift components (host, host:port, or URL). Scheme and port default to the --listen scheme and port. When unset, attempt to use the first public IPv4 non-loopback address registered on this host.")
 	flags.Var(&args.MasterPublicAddr, prefix+"public-master", "The master address for use by public clients, if different (host, host:port, or URL). Defaults to same as --master.")
+	flags.BoolVar(&args.OpenshiftEnabled, prefix+"openshift-enabled", false, "Controls if OpenShift additions are enabled or not.")
```

Comment:
Also, I wouldn't make this a command line arg. Do we expect atomic users to be running from the command line without a configuration file, or have an installer that sets up the config file for them? This feels like a config-only option to me. @deads2k?